test/typescript.js

```diff
@@ -52,8 +52,7 @@ describe('TypeScript', () => {
 					module: ts.ModuleKind.CommonJS,
 					allowJs: true,
 					configFilePath: undefined
-				},
-				compileOnSave: undefined
```

Comment:
Could you add an explanation to these lines in the description?

Reply:
I supposed that, but when I check [TS5.0 release notes](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-0.html) and current [tsconfig definition](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html), ([permalink](https://github.com/microsoft/TypeScript-Website/blob/9867af1a2a53c669c7a4d0be6dd1fa38a665bc47/packages/documentation/copy/en/project-config/tsconfig.json.md)), there were no occurrence of `compileOnSave`.
In contrary, it seems like a Visual Studio feature [see this link](https://www.typescriptlang.org/docs/handbook/asp-net-core.html).
So maybe it's not needed include it in our test?

Reply:
I thought it the opposite way. Nice catch!